src/avfoundation.cs

```diff
@@ -3945,6 +4067,42 @@ interface AVAssetWriter {
 		[iOS (8,0), Mac (10,10)]
 		[Export ("directoryForTemporaryFiles", ArgumentSemantic.Copy), NullAllowed]
 		NSUrl DirectoryForTemporaryFiles { get; set; }
+
+		// from category AVAssetWriterSegmentation (AVAssetWriter)
+
+		[TV (14, 0), NoWatch, Mac (11, 0), iOS (14, 0)]
+		[Export ("preferredOutputSegmentInterval", ArgumentSemantic.Assign)]
+		CMTime PreferredOutputSegmentInterval { get; set; }
+
+		[TV (14, 0), NoWatch, Mac (11, 0), iOS (14, 0)]
+		[Export ("initialSegmentStartTime", ArgumentSemantic.Assign)]
+		CMTime InitialSegmentStartTime { get; set; }
+
+		[TV (14, 0), NoWatch, Mac (11, 0), iOS (14, 0)]
+		[Export ("outputFileTypeProfile")]
+		[NullAllowed]
+		string OutputFileTypeProfile { get; set; }
```

Comment:
In ObjC that's `AVFileTypeProfile` (a `NSString` subclass) which you have defined before
Better bind this as a smart enum (or an `NSString`) but not a normal `string`